CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## Unreleased
+
+FEATURES
+* modules/gateway-task, modules/mesh-task, modules/dev-server, examples/mesh-gateways:
```

Comment:
nit: We haven't included changes to the examples in the changelog before.

In general, I think we shouldn't include examples in the changelog. What do you think?